export.py

```diff
@@ -1779,6 +1779,21 @@ def has_emissive_material(db):
     return False
 
 
+def export_for_bake(db):
+    export = True
+    node_names = []
+    for mat in db.material:
+        if mat and mat.node_tree:
+            nt = mat.node_tree
+            for n in nt.nodes:
+                node_names.append(n.name)
+    if node_names:
+        if 'PxrBakeTexture' not in node_names:
+            export = False
+
```

Comment:
Really?  One of the early tests I did didn't export a camera and I had an export fail because it couldn't be found in the instances list.  Or should I add a bake check to the export_camera line in write_rib?